core/update_system.py

```diff
@@ -425,8 +425,8 @@ def update_animation(cls, event: ev.AnimationEvent):
     def main_update(cls, tree: NodeTree, update_nodes=True, update_interface=True) -> Generator['SvNode', None, None]:
         """Thi generator is for the triggers. It can update outdated nodes and
```

Comment:
this -> this

Reply:
Done. [1805d42](https://github.com/nortikin/sverchok/pull/4612/commits/1805d421d516176edea2639444654bf7423b6b52)